requirements.txt

```diff
@@ -1,3 +1,4 @@
 terminaltables
 requests
 PyYAML
+packaging>=23.0
```

Comment:
@lgarber-akamai when installed through pip with `pip install linode-cli` this new dependency doesn't seem to be pulled. Isn't it necessary to add it here too e.g. [here](https://github.com/linode/linode-cli/blob/dev/setup.py#L78)?

Reply:
@okoriko Good catch! I'll put up a quick patch to fix this

Reply:
@okoriko This issue has been resolved in linode-cli `v5.34.1` :tada: